_data/utkarsht22/utkarsht22.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Utkarsh Thukral # No longer than 28 characters
+institution: Indian Institute of Technology Kanpur 🚩 # no longer than 58 characters
```

Comment:
I noticed you have a special char here - "the flag symbol". This could cause problems with your review, I would suggest to remove it.

Reply:
done.